sinks/influxdb/driver.go

```diff
@@ -188,55 +180,30 @@ func hashUID(s string) uint64 {
 	return h.Sum64()
 }
 
-func (sink *influxdbSink) storeEventNoColumns(event kube_api.Event) (*influxdb.Series, error) {
-	// Append labels to seriesName instead of adding extra columns
-	seriesName := strings.Replace(eventsSeriesName, "/", "_", -1)
-	labels := make(map[string]string)
-	if event.InvolvedObject.Kind == "Pod" {
-		labels[sink_api.LabelPodId.Key] = string(event.InvolvedObject.UID)
-		labels[sink_api.LabelPodName.Key] = event.InvolvedObject.Name
+func (sink *influxdbSink) StoreTimeseries(timeseries []sink_api.Timeseries) error {
+	if err := sink.createDatabase(); err != nil {
+		return err
 	}
-	labels[sink_api.LabelHostname.Key] = event.Source.Host
-	seriesName = fmt.Sprintf("%s_%s", util.LabelsToString(labels, "_"), seriesName)
-
-	columns := []string{}
-	columns = append(columns, "time")            // Column 0
-	columns = append(columns, "value")           // Column 1
-	columns = append(columns, "sequence_number") // Column 2
-
-	value, err := getEventValue(&event)
-	if err != nil {
-		return nil, err
-	}
-
-	// There's only one point per series for no columns
-	points := make([][]interface{}, 1)
-	points[0] = make([]interface{}, len(columns))
-	points[0][0] = event.LastTimestamp.Time.Round(time.Millisecond).Unix() // Column 0 - time
-	points[0][1] = sink.seqNum.Get(eventsSeriesName)                       // Column 1 - sequence_number
-	points[0][2] = value                                                   // Column 2 - value
-	return &influxdb.Series{
-		Name:    seriesName,
-		Columns: eventColumns,
-		Points:  points,
-	}, nil
-
-}
-
-func (self *influxdbSink) StoreTimeseries(timeseries []sink_api.Timeseries) error {
-	dataPoints := []*influxdb.Series{}
+	dataPoints := []influxdb.Point{}
```

Comment:
That doesn't work. Since we append and not replace entries, we end up with an array of twice the size. One option is to pass in a reference to individual elements and overwrite them. WDYT? Is it worth it?